backend/drive/drive.go

```diff
@@ -326,6 +326,14 @@ Photos folder" option in your google drive settings. You can then copy
 or move the photos locally and use the date the image was taken
 (created) set as the modification date.`,
 			Advanced: true,
+		}, {
+			Name:    "use_shared_date",
+			Default: false,
+			Help: `Use date file was shared instead of modified date.
+
+Note that, as with "--drive-use-created-date", this flag may have
+unexpected consequences when uploading/downloading files.`,
```

Comment:
Perhaps note here that `--drive-use-created-date` takes precedence if both set?

Reply:
Thanks for the feedback, that sounds good, will add that.